index.js

```diff
@@ -9,13 +9,16 @@ exports.buildInspectorUrl = buildInspectorUrl;
  * @param {number} inspectorPort as configured via --web-port
  * @param {number} debugPort as configured via --debug in the debugged app
  */
-function buildInspectorUrl(inspectorHost, inspectorPort, debugPort, fileToShow) {
+function buildInspectorUrl(inspectorHost, inspectorPort, debugHost, debugPort, fileToShow) {
```

Comment:
Ok. I added a branch to get debugHost from the last argument without changing function scheme.
```
  if (buildInspectorUrl.arguments.length > 4 && buildInspectorUrl.arguments[4]) {
      parts.query['host'] = buildInspectorUrl.arguments[4];
  }
```